### DISCUSSION TITLE
Get only reply part of the message?

### DISCUSSION THREAD

**Discussion by mastermaeco1993** [php-mime-mail-parser/php-mime-mail-parser#374] [Q&A]
Hello,

This plugin is awesome, I'm new to it and have one question. I have to parse reply emails, how can I get only reply content and not the whole email body?

**Comment by eXorus** [comment]
Sorry for the late answer.
You can find other php lib to parse reply emails. It's something very complicated because there are no standards.

**Comment by msphpd** [comment]
// use this method
preg_replace(
               '/\nOn(.*?)wrote:(.*?)$/si',
               "",
               $message->getTextContent() // your email body
           );
          Ref By  https://williamdurand.fr/EmailReplyParser/

**Comment by manulcaide1968** [comment]
nnn
